BREAKING.md

```diff
@@ -38,14 +39,21 @@ This field has been deprecated and will be removed in a future breaking change.
         fileName: string,
 -       createShareLinkType?: ShareLinkTypes,
 +       createShareLinkType?: ShareLinkTypes | ISharingLinkKind,
```

Comment:
unrelated?

Reply:
i got tricked by this to, that just github rendering the markdown in the breaking file

Reply:
Wow, it confused me as well. Yeah, this PR only changed the trailing space in line 41/42.